<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>t:Slim X2 Insulin Pump Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="screenStack">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="OffScreen" name="offScreen"/>
      <widget class="HomeScreen" name="homeScreen"/>
      <widget class="BolusScreen" name="bolusScreen"/>
      <widget class="SettingsScreen" name="settingsScreen"/>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="rightLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>t:Slim X2 Insulin
Pump Simulator</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="powerButton">
        <property name="text">
         <string>Power Button</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="changeBatteryLabel">
        <property name="text">
         <string>
Change Battery Level:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="batteryChangeLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLineEdit" name="batteryLevelInput">
          <property name="text">
           <string>100</string>
          </property>
          <property name="maxLength">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="batteryPercentUnitLable">
          <property name="text">
           <string>%  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setBatteryLevelButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="usbChargerButton">
        <property name="text">
         <string>Connect USB Charger</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="changeInsulinLabel">
        <property name="text">
         <string>
Change Stored Insulin Level:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="storedInsulinChangeLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="storedInsulinInput">
          <property name="text">
           <string>300</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="storedInsulinUnitsLabel">
          <property name="text">
           <string>u   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setStoredInsulinButton">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="changeCGMLabel">
        <property name="text">
         <string>
Change CGM (Continuous
Glucose Monitoring) Value</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="cgmLevelInput">
          <property name="text">
           <string>7.5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>mmol/L  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setCgmLevelButton">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="cgmConnectButton">
        <property name="text">
         <string>Disconnect CGM</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="physicalHomeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>173</width>
          <height>103</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgba(255, 255, 255, 0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/tandemlogo.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>173</width>
          <height>103</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>PASSWORD: 1234</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="debugLabel">
        <property name="text">
         <string>DEBUG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="debugOffScreenButton">
        <property name="text">
         <string>Off Screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="debugSettingsButton">
        <property name="text">
         <string>Settings Screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="debugBolusButton">
        <property name="text">
         <string>Bolus Screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="debugHomeButton">
        <property name="text">
         <string>Home Screen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="popupOverlayWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 150)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HomeScreen</class>
   <extends>QWidget</extends>
   <header>homescreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BolusScreen</class>
   <extends>QWidget</extends>
   <header>bolusscreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsScreen</class>
   <extends>QWidget</extends>
   <header>settingsscreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OffScreen</class>
   <extends>QWidget</extends>
   <header>offscreen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
